### ISSUE TITLE
Fix tabbing and focus during fullscreen

### ISSUE THREAD

**Issue by une-s** [une-s/KA-Graphic-Drawer#11]
Keyboard focus does not take fullscreen into consideration.
Only components within the fullscreen component should be focusable while a component is fullscreen. Currently other components can still grab keyboard focus.

Easiest fix:
Update Component.prototype.isVisible()
Return false if not in fullscreen component
